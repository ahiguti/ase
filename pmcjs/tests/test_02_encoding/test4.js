
function dump(txt) {
  for (var i = 0; i < txt.length; ++i) {
    print(txt.charCodeAt(i).toString(16));
  }
}

var txt = "あいうえお";
print("step1:");
dump(txt);
txt = Perl.TextToBin(txt);
print("step2:");
dump(txt);
txt = Perl.BinToText(txt);
print("step3:");
dump(txt);
