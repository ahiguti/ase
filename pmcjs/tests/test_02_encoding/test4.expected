step1:
3042
3044
3046
3048
304a
step2:
e3
81
82
e3
81
84
e3
81
86
e3
81
88
e3
81
8a
step3:
3042
3044
3046
3048
304a
